end-user-application-machines/new-agent-vm/Install-PCoIPAgent.ps1

```diff
@@ -88,9 +88,12 @@ Configuration InstallPCoIPAgent
                 $installerFileName = "SumoCollector_windows-x64_19_182-25.exe"
                 $uninstallerRegistryID = "7857-4527-9352-4688"  # This will need to change for every installer version change!
 
+				$sasToken = ($using:sasTokenAsCred).GetNetworkCredential().password
+				$blobLocation = ($using:sasTokenAsCred).GetNetworkCredential().username
+
                 $sumo_package = "https://teradeploy.blob.core.windows.net/binaries/$installerFileName"
-                $sumo_config = "$using:gitLocation/sumo.conf"
-                $sumo_collector_json = "$using:gitLocation/sumo-agent-vm.json"
+                $sumo_config = "$blobLocation/sumo.conf$sasToken"
```

Comment:
not a big deal if it works as is, but i think "${blobLocation}/sumo.conf${sasToken}" would be better

Reply:
changed it, you are right its clearer